newdle/api.py

```diff
@@ -187,7 +193,12 @@ def get_participant_busy_times(date, code, tz, participant_code=None):
     ).first_or_404('Specified participant does not exist')
     if participant.auth_uid is None:
         abort(422, messages={'participant_code': ['Participant is an unknown user']})
-    if not any(date == ts.date() for ts in participant.newdle.timeslots):
+    targetTz = timezone(tz)
+    newdleTz = timezone(participant.newdle.timezone)
+    if not any(
+        date == change_dt_timezone(ts, newdleTz, targetTz).date()
+        for ts in participant.newdle.timeslots
+    ):
```

Comment:
Hey, let's discuss this piece of code. You see, when updating the active date code to correctly reflect the active date on the answer calendar when dealing with 12 hour diff timezones (japan - sao paulo for example), I noticed a problem here. My test case is:

- Newdle on Sept 4th 10AM Japan
- I use Sao Paulo, so I see displayed Sept 3rd 10PM

In this case, I end up asking my busy slots for Sept 3rd, and this check would fail because all slots were for Sept 4th. The solutions I thought about were:

1. This solution I wrote: Cast the slots timezone before asking for the date. Depending on the demand of this particular endpoint, this could be a costly operation
2. Ask for busy time on the timezone and date of the newdle, and then cast and aggregate them on the frontend as I did in a previous commit we discussed
3. Remove this check and call `_get_busy_times` even if there are no `participant.newdle.timeslots`

What do you think we should do?

Reply:
I like your solution. It's simple, and timezone conversions are so cheap that they are perfectly fine. Every database query etc is likely more expensive than even a few timezone conversions.

Just one thing: We do not use camelCase in Python code - please use the more common `snake_case` in the code above.

Reply:
Ok, sounds good! And I just fixed the camelCase on Python, I was pretty tired when I wrote that code and the camelCase totally passed me by. Thanks for the catch!